Library/Formula/shc.rb

```diff
@@ -0,0 +1,23 @@
+class Shc < Formula
+  desc "Shell Script Compiler"
+  homepage "https://neurobin.github.io/shc"
+  url "https://github.com/neurobin/shc/archive/3.9.3a.tar.gz"
```

Comment:
@apjanke, the `chmod` call is what got me to this point to begin with (see outdated diff). =P